cmd/build/helmify/replacements.go

```diff
@@ -1,8 +1,11 @@
 package main
 
 var replacements = map[string]string{
-	"HELMSUBST_DEPLOYMENT_CONTAINER_RESOURCES": `
-{{ toYaml .Values.resources | indent 10 }}`,
+	"HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_CONTAINER_RESOURCES": `
+{{ toYaml .Values.controllerManager.resources | indent 10 }}`,
```

Comment:
these values needs to be added to `helmify/static/values.yaml`

Reply:
Added missing values `helmify/static/values.yaml`